spec/editors_draft.html

```diff
@@ -3298,6 +3298,231 @@ <h2>Evaluation Semantics</h2>
 
   </section>
 
+  <section id="import-export-composite-values">
+    <h1>Importing and Exporting of Composite Values</h1>
+
+    <section class="informative">
+      <h2>Motivation</h2>
+      <p>
+        In the normal handling of an RDF syntax, a blank node is scoped to the document, and any identifiers used in that document are merely a syntactic convenience for referencing blank nodes freshly allocated during parsing.
+        For example,
+      </p>
+        
+      <pre id="load-renaming-blank-example"
+         data-transform="updateExample"
+         data-content-type="text/turtle"
+         class="nohighlight example">
+      <!--
+      _:b1 ex:p4 "[ <http://example.org/alice>, <http://example.org/bob>, _:eve ]"^^cdt:List .
+      -->
+      </pre>
+      
+      <p>
+        Parsing this data as Turtle will result in a single triple whose subject is a blank node (now dissociated from the `b1` identifier), and whose object is a three-element <a>cdt:List literal</a>.
+        However, using a Turtle parser not aware of Composite Values will result in an object literal containing the substring "_:eve".
+        This presents a challenge, as allowing values to reference specific blank nodes by identifier in this way would allow multiple documents to reference the same blank node, and might expose <em>internal</em> blank node identifiers as <em>external</em> identifiers (a form of unintentional <a href="https://www.w3.org/TR/rdf11-concepts/#dfn-skolem-iri">skolemization</a>).
+      </p>
+        
+      <p>
+        The following sections describe requirements for conforming SPARQL systems that must be applied to composite values during their import into and export from datasets to support proper handling of blank nodes.
```

Comment:
Well, this spec of ours only defines interaction with these values in terms of:

* SPARQL 
* RDF Datasets

I suppose the latter could be used without SPARQL, so am open to language that would support that, but that feels a bit outside the scope of this spec (whose title starts with "SPARQL CDTs").

Reply:
I understand. Then, I am fine with using the more specific term "SPARQL system" instead of "RDF processing system."

Are you okay with the edit suggestions in the first comment of this thread?